api_app/analyzers_manager/file_analyzers/clamav.py

```diff
@@ -0,0 +1,27 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+from api_app.analyzers_manager.classes import FileAnalyzer, DockerBasedAnalyzer
+
+
+class ClamAV(FileAnalyzer, DockerBasedAnalyzer):
+    name: str = "ClamAV"
+    url: str = "http://static_analyzers:4002/clamav"
+    # interval between http request polling
+    poll_distance: int = 10
```

Comment:
`poll_distance` should be 3 seconds, `max_tries` should be 5 and `timeout` should be 15.